apollo-runtime/src/main/java/com/apollographql/apollo/cache/http/ResponseCacheStore.java

```diff
@@ -4,12 +4,35 @@
 
 import javax.annotation.Nonnull;
 
+/**
+ * ResponseCacheStore is an abstraction for a cache store that is used to read, modify or delete http responses.
+ */
 public interface ResponseCacheStore {
+  /**
+   * Returns ResponseCacheRecord for the entry named cacheKey or null if it doesn't exist is not currently readable.
+   *
+   * @param cacheKey the name of the entry
+   * @return ResponseCacheRecord
+   */
   ResponseCacheRecord cacheRecord(@Nonnull String cacheKey) throws IOException;
```

Comment:
Could you pls add
`@Nullable ResponseCacheRecord cacheRecord(@Nonnull String cacheKey) throws IOException;`
right @BenSchwab